File: 백준/Bronze/5341. Pyramids/Pyramids.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const result = [];
for(let i = 0; i < input.length - 1; i++) {
    const cur = input[i];
    
    let sum = 0;
    for(let j = 1; j <= cur; j++) {
        sum += j
    }
    
    result.push(sum);
}

console.log(result.join('\n'))

File: 백준/Bronze/5341. Pyramids/Pyramids.py
import sys
input_data = sys.stdin.read().strip().split()
numbers = [int(x) for x in input_data]

for i in range(len(numbers)):
    cur = numbers[i]
    
    if cur == 0:
        break;
    sum = 0
    for j in range(1, cur + 1):
        sum = sum + j
    
    print(sum)
